forge-gui/res/cardsfolder/s/sparas_bodyguard.txt

```diff
@@ -0,0 +1,15 @@
+Name:Spara's Bodyguard
+ManaCost: G W U
+Types: Creature Rhino Warrior
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoose | TriggerDescription$ When CARDNAME enters the battlefield, you may choose a creature card in your hand. If you do, it perpetually gains “This creature enters the battlefield with an additional shield counter on it.” Otherwise, put a shield counter on CARDNAME.
+SVar:TrigChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Creature.YouOwn | ChoiceTitle$ You may choose a creature card in your hand | RememberChosen$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualShield | Name$ Spara's Bodyguard's Perpetual Effect | Duration$ Permanent
```

Comment:
It would probably be good if the Effect is only created when you make a choice, so just add `ConditionDefined$ ChosenCard | ConditionPresent$ Card` to its line and your other cards that can use it :)